src/main/java/org/gelecekbilimde/scienceplatform/auth/model/request/RoleChangeRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.gelecekbilimde.scienceplatform.auth.model.request;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class RoleChangeRequest {
+
+	@NotNull
```

Comment:
```suggestion
	@NotBlank
```

Reply:
@meminkecik  bunun sebebi "a String field constrained with @NotBlank must be not null, and the trimmed length must be greater than zero."